docs/errors.md

```diff
@@ -164,6 +165,28 @@ Check the supported types for encrypted columns.
 
 
 
+<!-- ---------------------------------------------------------------------------------------------------- -->
+
+
+## Internal Mapper error <a id='mapping-internal-error'></a>
+
+An error that is internal to the EQL Mapper module. This may be a defect or a limitation of the current EQL Mapper.
```

Comment:
Not sure EQL Mapper is something we mention anywhere, but not sure what the customer-facing language should be. @kateandrews @auxesis any thoughts?

Reply:
@yujiyokoo @tobyhede maybe something like this?

"An internal error occurred when attempting to rewrite the SQL statement. This could be due to an internal invariant failure or because of a specific fragment of unsupported SQL syntax"

Reply:
@freshtonic that sounds good to me. I'll use that unless there's any other suggestion from other people